python/ray/dashboard/client/src/Dashboard.tsx

```diff
@@ -0,0 +1,487 @@
+import Link from "@material-ui/core/Link";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import createStyles from "@material-ui/core/styles/createStyles";
+import withStyles, { WithStyles } from "@material-ui/core/styles/withStyles";
+import SvgIcon, { SvgIconProps } from "@material-ui/core/SvgIcon";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Typography from "@material-ui/core/Typography";
+import React from "react";
+import { Route } from "react-router";
+import { Link as RouterLink } from "react-router-dom";
+import Errors from "./Errors";
+import Logs from "./Logs";
+import UsageBar from "./UsageBar";
+
+const formatByteAmount = (amount: number, unit: "mebibyte" | "gibibyte") =>
```

Comment:
let's move `format*` functions to a `Utils.tsx`

Reply:
Leaving this for future refactoring if needed, as per our offline discussion.